packages/jbrowse-plugin-apollo/src/FeatureDetailsWidget/ApolloTranscriptDetailsWidget.tsx

```diff
@@ -112,6 +119,45 @@ export const ApolloTranscriptDetailsWidget = observer(
       props,
     ) as React.ElementType<CustomComponentProps>
 
+    const BiotypesComponent = pluginManager.evaluateExtensionPoint(
+      'Apollo-BiotypesComponent',
+      () => null,
+      {
+        key: 'status',
+        feature,
+        session,
+        assembly,
+        TypeChange,
+        DeleteFeatureChange,
+        AddFeatureChange,
+        FeatureAttributeChange,
+      },
+    ) as React.ElementType
+
+    const SaveComponent = pluginManager.evaluateExtensionPoint(
+      'Apollo-HavanaSaveComponent',
+      () => null,
+      {
+        key: 'status',
+        feature,
+        session,
+        assembly,
+        token,
+      },
+    ) as React.ElementType
+
+    const HavanaAttributesComponent = pluginManager.evaluateExtensionPoint(
+      'Apollo-HavanaAttributes',
```

Comment:
Let's name the extension points by where they appear instead of having a custom Havana name, e.g. `"Apollo-TranscriptDetailsLocationCustomComponent"`